configure.ac

```diff
@@ -57,9 +55,6 @@ CXXFLAGS="-DIN_LIBOFX $CXXFLAGS"
 
 LIBOFX_VERSION=$LIBOFX_MAJOR_VERSION.$LIBOFX_MINOR_VERSION.$LIBOFX_MICRO_VERSION
 
-AC_SUBST(LIBOFX_MAJOR_VERSION)
-AC_SUBST(LIBOFX_MINOR_VERSION)
-AC_SUBST(LIBOFX_MICRO_VERSION)
```

Comment:
The changes here are totally understandable. I'll cherry-pick them already.

Reply:
Thanks!